AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -234,6 +234,18 @@ def create_outfile(output_dirpath, logs_date, pkg_manager):
             outfile.write("--------------------------------------------------------------------------------\n")
         outfile.write("--------------------------------------------------------------------------------\n")
 
+        # du output on events folder
+        for flag in ["", "--apparent-size"]:
+            du_full_cmd = DU_CMD.format(flag)
+            outfile.write("Output of command: {0}\n".format(du_full_cmd))
+            outfile.write("========================================\n")
+            outfile.write(helpers.run_cmd_output(du_full_cmd))
+            outfile.write("--------------------------------------------------------------------------------\n")
```

Comment:
Why not match the starting line format ======= ?

Reply:
The `=` line separates the command and its output, while the `-` line separates different commands; example below:

```
Output of command: ls -l
========================================
-rw-rw-r-- 1 aswatt aswatt 21275 Apr  4 16:22 ama_tst.tgz
--------------------------------------------------------------------------------
Output of command: pwd
========================================
/home/aswatt
--------------------------------------------------------------------------------```

I was trying to make it the most legible, but lmk if you think there's a better way to format this!